tidb-cloud/scale-tidb-cluster.md

```diff
@@ -109,9 +109,12 @@ For a running cluster, AWS and Google Cloud do not allow in-place storage capaci
 
 ## Increase node size
```

Comment:
Release note:
Support [increasing the Node Size](https://docs.pingcap.com/tidbcloud/scale-tidb-cluster#increase-node-size)(vCPU + RAM), aka scale-up, for TiDB, TiKV, and TiFlash node of clusters created after November 30, 2022.